uilib/src/components/Loader/index.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { Svg, Circle, G, Rect } from "react-native-svg";
+import { useTheme } from "styled-components/native";
+import { TouchableOpacity } from "react-native";
+
+type Props = {
+  // float number between 0 and 1
+  progress?: number;
+
+  // function triggered when pressing the loader
+  onPress: () => void;
+
+  // Display the square in the middle of the loader
+  displayCancelIcon?: boolean;
+};
+
+const R = 25;
```

Comment:
What does `R` stands for here? Could you provide a more precise / specific name?

If it represents a radius, a better name could be `RADIUS`

Reply:
Part of the old loader code (progressCircle), it's fixed, thanks